src/main/java/mate/academy/springboot/swagger/model/Product.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.springboot.swagger.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data

```

Comment:
It isn't good to use @Data with entities. (bad practices - can make mistakes while working with equals/hash)
https://habr.com/ru/company/haulmont/blog/564682/